components/select/style/index.less

```diff
@@ -401,7 +401,8 @@
     display: none;
   }
 
-  background-color: white;
+  background-color: @component-background;
+  border: 1px solid @border-color-base;
```

Comment:
sorry. it's added by rebase automaticlly. so I think it may be from other commits. after check. I will remove it.